src/views/order_view.js

```diff
@@ -0,0 +1,70 @@
+import Backbone from 'backbone';
+import Order from '../models/order';
+
+const OrderView = Backbone.View.extend({
+  initialize(params) {
+    this.bus = params.bus;
+    this.template = params.template;
+    this.triggered = false;
+
+    const eventName = `${this.model.get('symbol').toLowerCase()}_change`;
+    this.listenTo(this.bus, eventName, this.checkPrice, this.model);
```

Comment:
I like that each `OrderView` is listening for a specific event, but I think we can go a step further in simplifying this design. When you create the `Order` above in `app.js`, you need to know what `Quote` it corresponds to so you can check the current stock price. You could make the `Quote` an attribute of the `Order`, and then have the `OrderView` (or even the `Order` itself) listen directly to the `Quote` for price changes, cutting the bus out entirely.